src/Handler/EasyHandle.php

```diff
@@ -35,7 +35,7 @@ final class EasyHandle
     /** @var int cURL error number (if any) */
     public $errno = 0;
 
-    /** @var \Throwable Exception during on_headers (if any) */
+    /** @var ?\Throwable Exception during on_headers (if any) */
```

Comment:
`Throwable|null`

Reply:
@rvanlaak  there are other places as well.. so pls check them out.

Reply:
fixed in 583f8291db35249b94ca8893068002a654caf4a1